src/platform/ESP32/ConfigurationManagerImpl.cpp

```diff
@@ -57,14 +58,7 @@ uint32_t ConfigurationManagerImpl::mTotalOperationalHours = 0;
 
 void ConfigurationManagerImpl::TotalOperationalHoursTimerCallback(TimerHandle_t timer)
 {
-    mTotalOperationalHours++;
-
-    CHIP_ERROR err = ConfigurationMgrImpl().StoreTotalOperationalHours(mTotalOperationalHours);
-
-    if (err != CHIP_NO_ERROR)
-    {
-        ChipLogError(DeviceLayer, "Failed to store total operational hours: %" CHIP_ERROR_FORMAT, err.Format());
-    }
+    (void) ConfigurationMgrImpl().StoreTotalOperationalHours(++mTotalOperationalHours);
```

Comment:
why is the CHIP_ERROR ignored? Should we at least log it?

Reply:
ah, the whole point of the PR ... in this case we should have a comment explaining this is done on purpose and why.

Reply:
Added the comment!